modules/API/pages/gsql-endpoints.adoc

```diff
@@ -3144,6 +3149,9 @@ This endpoint is used to intepret query. Please note that calling this endpont n
 | `graph`
 | yes
 | the query interpreted under which graph
+| `queue`
+| optional
+| the workload queue 
```

Comment:
```suggestion
```
Do not add to this table.  A header option is different than a parameter. Our REST API uses HTTP (the almost universal choice).  An **HTTP parameter** is placed in the query string, at the **end of the 
 URL address**: https://en.wikipedia.org/wiki/Query_string.
 An **HTTP header** is not part of the URL but a part of the **metadata that describes the format or handling** of the request: https://en.wikipedia.org/wiki/List_of_HTTP_header_fields